packages/reports/addon/templates/components/dimension-selector.hbs

```diff
@@ -18,7 +18,7 @@
     {{#if item.dateTimeDimension}}
       {{#if (feature-flag "enableRequestPreview")}}
         <div class="grouped-list__item-container {{if this.selectedTimeGrain "grouped-list__item-container--selected grouped-list__item-container--disabled"}}">
-          <span class="grouped-list__item-label"
+          <button class="grouped-list__item-label"
```

Comment:
type="button"

Maybe double check the focus in firefox issue with this?

Reply:
Doesn't fix the firefox issue it seems, but I've added it.
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/button#Clicking_and_focus

The animation stopped working because I decided, being the genius that I am, to shorten the `blur` callback I passed to the event listener. That broke it, but it should be working now if you want to try again.

Reply:
Missed type=button for this instance